sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs

```diff
@@ -6,13 +6,17 @@
 namespace Azure.AI.FormRecognizer
 {
     /// <summary>
-    /// The set of options that can be specified when calling a recognize content method
-    /// to configure the behavior of the request.
+    /// The set of options that can be specified when calling a Recognize Content method
+    /// to configure the behavior of the request. For example, specify the content type of the
+    /// form, the language of the form, and the custom page numbers in a multi-page document to analyze.
     /// </summary>
     public class RecognizeContentOptions
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="RecognizeContentOptions"/> class.
+        /// Initializes a new instance of the <see cref="RecognizeContentOptions"/> class which
+        /// allows to set options that can be specified when calling a Recognize Content method
+        /// to configure the behavior of the request. For example, specify the content type of the
+        /// form, the language of the form, and the custom page numbers in a multi-page document to analyze.
```

Comment:
If you decide to change the class description, just a reminder to update this one as well.